DeepLinkDemo/DeepLinkDemo/Controllers/HomeViewController.swift

```diff
@@ -65,28 +71,37 @@ class HomeViewController: UITableViewController {
         CommonMethod.sharedInstance.contentMetaData = nil
         reachability = Reachability()!
         reachability!.whenReachable = { reachability in
-            
+            Branch.setBranchKey("key_test_om2EWe1WBeBYmpz9Z1mdpopouDmoN72T")
```

Comment:
Is this change to allow no tracking changes prior to first events?

Reply:
Right.